policy/modules/services/opensm.if

```diff
@@ -0,0 +1,86 @@
+## <summary>Policy for opensm</summary>
```

Comment:
Please put a description for opensm. It should be obvious this is a policy for opensm.